### PR TITLE
Fix games embedding format

### PR DESCRIPTION
**Pull request by lukaswajdzik**

### CODE REVIEW

File: pkg/app/command/compute_game_vectors.go [hunk 1]
```diff
@@ -77,11 +79,10 @@ func renderGame(
 	gameTagsVector []float64,
 ) string {
 	gameStrings := []string{
-		game.Id,
-		game.Name,
+		strings.Join([]string{game.Id, game.Name}, " "),
```
Comment (krzysztofreczek) [inline pkg/app/command/compute_game_vectors.go:82]: It is possible that the name of a game contains a comma. If yes, then the CSV structure will be broken. I guess we should sanitize the game name and remove all commas if there are any.

File: pkg/app/command/compute_track_vectors.go [hunk 2]
```diff
@@ -82,7 +84,7 @@ func renderTrack(
 		track.Title,
 		renderTrackVector(trackVector),
 	}
-	return strings.Join(trackStrings, utils.Delimiter)
+	return strings.Join(trackStrings, trackDelimiter)
```
Comment (krzysztofreczek) [inline pkg/app/command/compute_track_vectors.go:87]: Maybe we should follow the same rules as in the other command here? Let's use a comma delimiter here and join ID, Artist, and Title into one column. The pattern could be "{ID}: {Artist} - {Title}". And again, we will need to remove commas from both Artist and Title strings.

Comment (krzysztofreczek) [reply pkg/app/command/compute_track_vectors.go:87]: Might be there are some tests checking that logic so you will need to adjust it.

File: pkg/app/command/compute_game_vectors.go (hunk 1 above)
Comment (lukaswajdzik) [reply pkg/app/command/compute_game_vectors.go]: done

File: pkg/app/command/compute_track_vectors.go (hunk 2 above)
Comment (lukaswajdzik) [reply pkg/app/command/compute_track_vectors.go]: done

File: pkg/app/command/compute_track_vectors.go [hunk 3]
```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/nhood-org/nhood-engine-utils/pkg/model"
 )
 
+const trackDelimiter = ":|:"
+
```
Comment (krzysztofreczek) [inline pkg/app/command/compute_track_vectors.go:13]:
```suggestion
```

File: pkg/app/command/compute_game_vectors.go [hunk 4]
```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/nhood-org/nhood-engine-utils/pkg/model"
 )
 
+const gameDelimiter = ","
+
```
Comment (krzysztofreczek) [inline pkg/app/command/compute_game_vectors.go:13]:
```suggestion
```

**Review by krzysztofreczek — APPROVED** [review]

**Pull request merged**